plugin/storage/badger/spanstore/read_write_test.go

```diff
@@ -587,14 +588,14 @@ func BenchmarkServiceIndexLimitFetch(b *testing.B) {
 
 // Opens a badger db and runs a test on it.
 func runLargeFactoryTest(tb testing.TB, test func(tb testing.TB, sw spanstore.Writer, sr spanstore.Reader)) {
+	assert := require.New(tb)
```

Comment:
Please don't do that, it's making tests non-intuitive and breaks the pattern across the project. `assert` and `require` should be used explicitly as needed, `require` if test cannot proceed further, `assert` if it can.

Reply:
I don't see a pattern across tests in the project sadly regarding that. But I'll make it assert.New(t) then, which seems to be the common pattern (regardless of the actual behavior wanted).